tests/main.test.js

```diff
@@ -69,14 +69,14 @@ test("Subdirectories", async ({ page, prefix, baseUrl }) => {
 	await expect(await page.getByText("Docs").getAttribute("href")).toEqual(prefix + "/docs/");
 	await expect(await page.getByText("Homepage").getAttribute("href")).toEqual(prefix || "/");
 
-	await page.getByText("b").click();
+	await page.getByText("b", { exact: true }).click();
```

Comment:
i can understand why this change.. will query inside [baseElement](https://testing-library.com/docs/react-testing-library/api#baseelement) but you use page object so [exact: true](https://testing-library.com/docs/queries/bytext/) i guess is a way to dont choose another different than the exact href to test? i dont know about such parameter for, the "b" is too ambiguous or generic but "page" must be enough to difference it

Reply:
Previously, we matched the character B anywhere in the page, which is too permissive, and why it broke in this PR.
The proposed fix is to make the text match exact, which means the anchor should exactly be equal to B.